### ISSUE TITLE
Add build scripts

### ISSUE THREAD

**Issue by ocramz** [redneb/ghc-alt-libc#4]
Hi, could you show the instructions that were used to compile these releases? Thank you!

**Comment by redneb** [comment]
Hi,

If you are talking about how I compile each version of GHC when it comes out, then the answer is that I mostly follow the [standard](https://gitlab.haskell.org/ghc/ghc/wikis/making-releases#making-the-binary-builds) process for building GHC. If there is any need to diverge from those instructions, I document that in the [release notes](https://github.com/redneb/ghc-alt-libc/blob/master/NOTES-musl.md#release-notes) that I've included in this repository. So for e.g. for 8.6.5 I did something like this:

    $ cat > mk/build.mk <<EOF
    V=1
    HADDOCK_DOCS=NO
    HSCOLOUR_SRCS=NO
    BUILD_SPHINX_HTML=NO
    BUILD_SPHINX_PDF=NO
    BeConservative=YES
    EOF
    $ ./configure --disable-ld-override
    $ make
    $ make binary-dist

I ran the above in a gentoo musl system where I had previously installed the previous version of GHC.

This previous version of GHC was build by following a similar process relying on an older version of GHC, and so on, all the way back to when I initially bootstraped GHC on musl. That happened 4 years ago and I explained the process for that in my [reddit post](https://www.reddit.com/r/haskell/comments/37m7q7/ghc_musl_easier_static_linking/) (and specifically [this](https://www.reddit.com/r/haskell/comments/37m7q7/ghc_musl_easier_static_linking/crnytml) comment) where I announced this project. Of course, the contents of this last paragraph are of historical interest only, you don't need to follow these instructions to produce GHC binaries any more.

Hope that helps.
